scholia/app/templates/work_cito_intention.html

```diff
@@ -0,0 +1,37 @@
+{% extends "base.html" %}
+
+
+{% block scripts %}
+{{super()}}
+
+<script type="text/javascript">
+  citoIncomingSparql = `
+SELECT ?citingArticle ?citingArticleLabel ?intention ?intentionLabel WHERE {
+  VALUES ?CITEDARTICLE { wd:{{ q }} }
+  VALUES ?intention { wd:{{ q2 }} }
+  ?citingArticle p:P2860 ?citationStatement .
+  ?citationStatement pq:P3712 ?intention ;
+                     ps:P2860 ?CITEDARTICLE .
+  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
+}`
```

Comment:
This should be in an external .sparql file (so we don't regress on #1284)

Reply:
Ah, right! Yeah, I had planned on doing that, but forgot it in the excitement :)

Reply:
Egon, I'm working in renaming all HTML files, changing the underscore by hyphen (https://github.com/WDscholia/scholia/pull/1558), according to a recommendation made by [Finn](https://github.com/WDscholia/scholia/pull/1487#issuecomment-878322478) in #1284. Maybe the HTML and SPARQL filename should use hyphen instead of underscore

Reply:
Fixed the filename and the external SPARQL now.